packages/synapse-react-client/src/lib/containers/synapse_form_wrapper/SynapseFormSubmissionsGrid.tsx

```diff
@@ -453,9 +455,9 @@ export default class SynapseFormSubmissionGrid extends React.Component<
           {this.renderLoading(this.props.token, this.state.isLoading)}
           {this.renderUnauthenticatedView(this.props.token)}
 
-          {!this.state.isLoading && (
+          {this.props.token && !this.state.isLoading && (
```

Comment:
Ensure that when toggling `isAuthenticated` in Storybook, the "logged in" section of the component only appears if actually logged in